pages/calendar.vue

```diff
@@ -4,43 +4,86 @@
     justify-center
     align-center
   >
-    <categories-bar />
+    <categories-bar
+        @nextPressed="$refs.calendar.next()"
+        @prevPressed="$refs.calendar.prev()"
+     />
     <v-calendar
+        v-model="start"
         class="calendar"
+        :events="calendarEvents"
+        event-overlap-mode="stack"
+        :event-color="getEventColor"
+        event-text-color="black"
+        ref="calendar"
     />
   </v-layout>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
 import CategoriesBar from '@/components/CalendarComponents/categoriesBar.vue';
-import { allEvents } from '@/queries/Events';
+import { AllEvents } from '@/queries/Events';
 import Event from '@/models/Event.ts';
 
+interface CalendarEvent {
+    name: string; 
+    start: string;
+    end: string;
+    color: string;
+};
+
 export default Vue.extend({
+    data() {
+        return {
+            start: ''
+        };
+    },
     components: {
         CategoriesBar
     },
     apollo: {
         allEvents: {
-            query: allEvents
+            query: AllEvents
         }
     },
     computed: {
         events() : Event[] {
-            return this.$data.allEvents.nodes.map((event: any) => {
-                return {
-                    id: event.id,
-                    startDate: event.startDate,
-                    endDate: event.endDate,
-                    title: event.title,
-                    category: {
-                        id: event.category.id,
-                        title: event.category.title,
-                        color: event.category.color
+            if (this.$data.allEvents !== undefined) {
+                return this.$data.allEvents.nodes.map((event: any) => {
+                    return {
+                        id: event.id,
+                        startDate: event.startDate,
+                        endDate: event.endDate,
+                        title: event.title,
+                        category: {
+                            id: event.category.id,
+                            title: event.category.title,
+                            color: event.category.color
+                        }
                     }
+                });
+            } else {
+                return [];
+            }
+        }, 
+        calendarEvents(): CalendarEvent[] {
+            return this.events.map((event: Event) => {
+                return {
+                    name: event.title, 
+                    start: this.formatDate(new Date(event.startDate)),
```

Comment:
Use "startDate" as  event variable

Reply:
Calendar requires start and not startDate